### PR TITLE
pdf-watermark

### PR DESCRIPTION
**Pull request by madihamallick**
# Description

Adding watermark to pdf file pages

## Related Issues or Pull Requests

#17 

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

### CODE REVIEW

File: pdf-watermark/pdf-watermark.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+from PyPDF2 import PdfFileWriter
+from PyPDF2 import PdfFileReader
+
+
+ori_pdf = PdfFileReader('inputfile.pdf')
+mark = PdfFileReader('watermark2.pdf')
+
```
Comment (avinashkranjan) [inline pdf-watermark/pdf-watermark.py:7]:
```
ori_pdf = PdfFileReader('inputfile.pdf')
mark = PdfFileReader('watermark2.pdf')
```

**Change this code to this to make it more convenient for the user**

```
input_File = input(
    "Enter File Name You want to add Watermark too[Include .pdf extension]: ")
watermark = input("Enter Watermark File Name [Include .pdf extension]: ")

ori_pdf = PdfFileReader(input_File)
mark = PdfFileReader(watermark)
```

File: pdf-watermark/pdf-watermark.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+watermark = mark.getPage(0)
+original_pdf = ori_pdf.getPage(0)
+
+ori_page_count = ori_pdf.getNumPages()
+
+output = open('finalfile.pdf','wb')
```
Comment (avinashkranjan) [inline pdf-watermark/pdf-watermark.py:14]: Change `finalfile.pdf` to `watermarked.pdf` for user convenience

**Review by avinashkranjan — CHANGES_REQUESTED** [review]
@madihamallick Please make the following changes

File: pdf-watermark/pdf-watermark.py (hunk 2 above)
Comment (madihamallick) [reply pdf-watermark/pdf-watermark.py]: done

**Review by avinashkranjan — APPROVED** [review]
LGTM 💯

**Comment by avinashkranjan** [comment]
@all-contributors please add @madihamallick  for code

**Pull request merged**